tests/Integration/Mail/RenderingMailWithLocaleTest.php

```diff
@@ -31,14 +31,14 @@ public function testMailableRendersInDefaultLocale()
     {
         $mail = new RenderedTestMail;
 
-        $this->assertEquals("name\n", $mail->render());
+        $this->assertRegExp('/name\r?\n/', $mail->render());
```

Comment:
This actually weakens the tests and is prone to false negatives. I'd suggest trying `PHP_EOL` instead.